doc/BorderRouter.rst

```diff
@@ -1,182 +1,220 @@
 forwarding the packets over the network that it receives from the processing routines. It collects
 packets from the go channel up to the point where either no further packet is available or the batch
 size is reached.
 Then it forwards the packets as a batch.
-Afterwards it returns each buffer to the receiver from which it originates.
+Afterwards it returns the buffers back to the receivers.
 
-![Border Router Design](fig/border_router/br_design.png)
+.. image:: fig/border_router/br_design.png
 
-### Mapping of processing routines
+Mapping of processing routines
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To prevent any packet reordering on the fast-path, we map the tuple of source and flowID, see the
-[SCION header documentation](https://github.com/scionproto/scion/blob/master/doc/protocols/scion-header.rst),
-to a fixed processing routine using a hash function together with a random value which is generated
+To prevent any packet reordering on the fast-path, we map the triple of source (ISD-AS + local address),
+destination (ISD-AS + local address) and flowID, see the
+`SCION header documentation <https://github.com/scionproto/scion/blob/master/doc/protocols/scion-header.rst>`_
+to a fixed processing routine using the fnv-1a hash function together with a random value which is generated
 on startup to prevent pre-computations of the exact mapping.
 
-### Slow path
+Slow path
+^^^^^^^^^^^
 
 During processing, packets that have to follow the slow path are identified and forwarded to the
 slow-path processing routines.
+To do so, we hand over the current buffer to the slow-path routine together with all the information
+that the processing routine already computed that are required by the slow-path processing routine.
 Rate limiting of slow-path operations is not implemented explicitly, but only implictily through
 specifying the number of slow-path processing routines in the configuration.
 In case a packet is identified to belong to the slow path but the queue of the slow path is full, the
 packet is dropped.
 Packets currently identified for slow-path are:
 
-* Malformed packets
+- Malformed packets
 
-* SCMP traceroute packets
+- SCMP traceroute packets
 
-## Configuration
+Configuration
+---------------
 
 The configuration of the border router will remain in the border router toml file.
 The following configuration entries are added:
 
-### Pool size
+Pool size
+^^^^^^^^^^^
 
-Since a pool of packet buffers is bound to a receiver, the size of the pool can be configured for each
-receiver seperately.
-It makes sense to make this configurable on a per border router interface level because not every
-interface might have the same expected load and hence not every interface requires the same pool size.
+The size of the packet buffer pool of the receivers can be configured.
 An optimal value should be derivable from the maximum latency of processing and forwarding a packet,
 the speed of the network interfaces and the number of available CPU cores.
```

Comment:
Maybe add the computation that Matthias did?

Reply:
Then I guess we don't need the previous sentence anymore?
("should be derivable from the maximum latency...")

Also: an optimal could -> an optimal value could